allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_data_transforms.py

```diff
@@ -243,8 +244,54 @@ def get_stimulus_templates(self) -> Optional[StimulusTemplate]:
         -------
         StimulusTemplate or None if there are no images for the experiment
         """
+
+        # TODO: Eventually the `grating_images_dict` should be provided by the
```

Comment:
Correct, it's also the reason why I didn't want to give `grating_images_dict` a default value to be filled in by the `get_stimulus_templates()` function.

Reply:
Does it at least need to have a default value of `None` so that, when processing images, `get_stimulus_templates()` does not expect a `grating_images_dict`?